### PR TITLE
feat(RTL): rtl/ltr build

### PR DESCRIPTION
**Pull request by jacekkarczmarczyk**
## Description
- a la carte: removes rtl styles in ltr-only mode and vice versa
- dist: creates a ltr-only and rtl-only build

If it's approved and merged I think we can close https://github.com/vuetifyjs/vuetify/issues/262 as all components and framework in general fully supports RTL, there are still some bugs but they are tracked in separate issues

### a la carte
1. create a new project with vue-cli, add vuetify-plugin
2. https://next.vuetifyjs.com/en/framework/theme#setup-vueconfigjs
3. `yarn add css-mqpacker autoprefixer -D`
4. bidi: `yarn build`
5. ltr-only: add `$use-rtl: false` to `main.sass`, run `yarn build`
6. rtl-only: add `$use-ltr: false` to `main.sass`, run `yarn build`

### dist
1. `yarn build:dist`

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

## Checklist:

- [x] The PR title is no longer than 64 characters.
- [x] The PR is submitted to the correct branch (`master` for bug fixes and documentation updates, `dev` for new features and breaking changes).
- [x] My code follows the code style of this project.
- [ ] I've added relevant changes to the documentation (applies to new features and breaking changes in core library)

### CODE REVIEW

File: packages/vuetify/build/config.js [hunk 1]
```diff
@@ -6,6 +6,7 @@ const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin')
 
 const base = require('./webpack.prod.config')
 const version = process.env.VERSION || require('../package.json').version
+const dirPostfix = process.env.VUETIFY_DIRECTION ? `.${process.env.VUETIFY_DIRECTION}` : ''
```
Comment (sh7dm) [inline packages/vuetify/build/config.js]: maybe `-rtl`/`-ltr`?
```suggestion
const dirPostfix = process.env.VUETIFY_DIRECTION ? `-${process.env.VUETIFY_DIRECTION}` : ''
```

Comment (jacekkarczmarczyk) [reply packages/vuetify/build/config.js]: I think dot is a common convention, not attached to it though, if more people want dash i'm ok. Resolving for now

**Comment by nekosaur** [comment]
Is there a @vuetifyjs/core-team decision to be made here? If not, it seems like the faster this gets merged the better.

**Comment by jacekkarczmarczyk** [comment]
There's one thing I'd like to be done here - now there's full build running to get the rtl/ltr/bidi CSS, but I guess the build could be run just once and then postcss or other processor could make rtl/ltr version just by converting bidi version, that could speed up the process significantly

**Comment by johnleider** [comment]
Just needs docs and we can merge it in.

**Pull request closed without merging**